src/backend/parser/cypher_clause.c

```diff
@@ -270,9 +279,15 @@ static cypher_target_node *get_referenced_variable(ParseState *pstate,
 
 //call...[yield]
 static Query *transform_cypher_call_stmt(cypher_parsestate *cpstate,
-                                      cypher_clause *clause);
+                                         cypher_clause *clause);
 static Query *transform_cypher_call_subquery(cypher_parsestate *cpstate,
-                                          cypher_clause *clause);
+                                             cypher_clause *clause);
+
+//call...[yield]
```

Comment:
the function headers `transform_cypher_call_stmt` and `transform_cypher_call_subquery` are duplicate below.

Reply:
Weird that the compiler didn't flag that.